src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,51 @@
 package core.basesyntax;
 
-class Pair {
+/*реализуйте generic-класс Pair, содержащий пару элементов разных
+типов и не запрещающий элементам принимать значение null.
+Реализуйте методы getFirst(), getSecond(), equals() и hashCode(),
+а также статический фабричный метод of().
+Конструктор должен быть закрытым (private).
+С корректно реализованным классом Pair должен компилироваться и успешно работать следующий код:*/
 
+class Pair<A, N> {
+    private A age;
```

Comment:
it can get very confusing if you pass here let's say "Car" class, or "Building".
it'll be better if use more general naming